### DISCUSSION TITLE
posts/pulumi-vs-terraform

### DISCUSSION THREAD

**Discussion by giscus** [TechWatching/techwatching.dev#31] [Announcements]
# posts/pulumi-vs-terraform

https://www.techwatching.dev/posts/pulumi-vs-terraform

**Comment by eriklz** [comment]
Very nice write-up, good work!

**Reply by TechWatching** [reply]
Thanks! Glad you enjoyed 😀.

**Comment by spopida** [comment]
Excellent, thorough and convincing article - thanks!.

**Reply by TechWatching** [reply]
Thanks 👍
